sihl/src/index.mld

```diff
@@ -834,6 +835,99 @@ todo.add
 user.admin
 v}
 
+{2 Generators}
+
+Sihl has built-in generators [gen.service], [gen.view] and [gen.html]. Similar to Rails they help you generating resources, services, repositories and HTML views. Unlike Rails, Sihl doesn't have an abstraction layer for data access (such as an ORM). You should be familiar with SQL.
+
+The generators create files and folders before printing instructions for manual steps to finalize the generation. [gen.html] can create a fully functional CRUD web interface from a schema.
+
+Generators are CLI commands just like [migrate], run the generator commands without any arguments to display help text describing its usage.
+
+{3 Service}
+
+{v
+$ make sihl gen.service mariadb order order_number:int premium:bool price:int location:string delivery_date:datetime
+2021-04-12T16:20:46-00:00 [INFO] [sihl.core.app]: Setting up...
+2021-04-12T16:20:46-00:00 [INFO] [sihl.core.configuration]: SIHL_ENV: development
+Wrote file '/home/josef/src/sihl-demo/app/context/order/order.ml'
```

Comment:
Are you sure you want your path in the docs?

Reply:
Thanks, removed :)